content/ember/v3/jquery-apis.md

```diff
@@ -0,0 +1,99 @@
+---
+id: jquery-apis
+title: Remove jQuery APIs
```

Comment:
```suggestion
title: Replace jQuery APIs
```

Reply:
I can do that, by why do you think it's better? As we don't introduce any new APIs as a replacement, we are actually only removing some.

Reply:
I checked existing deprecations and the phrasing seems to generally be the action that you need to do. While we did not introduce new APIs, we suggest that people use `this.element` and DOM APIs because as of 3.0 they are present in browsers supported by Ember.
It's fine either way!